DemoApp/ios/DemoApp/AppDelegate.m

```diff
@@ -25,7 +25,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
   [AppCenterReactNativePush register];  // Initialize AppCenter push
 
-  [AppCenterReactNativeCrashes registerWithAutomaticProcessing];  // Initialize AppCenter crashes
+  [AppCenterReactNativeCrashes register];  // Initialize AppCenter crashes
```

Comment:
Is register manual processing? Really those ios methods are poorly named I am always confused xD

Reply:
I think so? this wasn't a manual change

Reply:
Did you test attachments? and user confirmation?

Reply:
I was able to send attachments

Reply:
and i got the user confirmation pop up

Reply:
then @TroubleMakerBen did not test that last time as automatic process was enabled before, no way it could have worked